app/services/participants/withdraw/validate_and_change_state.rb

```diff
@@ -5,11 +5,18 @@ module Withdraw
     module ValidateAndChangeState
       extend ActiveSupport::Concern
       include ActiveModel::Validations
+      REASONS = %w[
+        left-teaching-profession
+        moved-school
+        mentor-no-longer-being-mentor
+        school-left-fip
+        other
+      ].freeze
```

Comment:
Withdraw reasons certainly are not the same between ecf and npq. Does this change not cause NPQ tests for withdrawals to fail? Or does the constant included in the npq module override this constant?